docs/riff_system_uninstall.md

```diff
@@ -0,0 +1,37 @@
+## riff system uninstall
+
+Remove all riff and Knative system components
```

Comment:
"all" is a bit misleading. We make a good faith attempt to remove everything, but cannot promise to get everything. Particularly as knative releases evolve with different names. 

This is one area where helm has an advantage by sticking an annotation on all resources it creates.